packages/node_modules/pouchdb-core/src/constructor.js

```diff
@@ -24,11 +24,24 @@ function prepareForDestruction(self) {
   }
 
   function onConstructorDestroyed() {
-    self.removeListener('destroyed', onDestroyed);
+    onClosed();
     self.emit('destroyed', self);
   }
 
+  function onClosed() {
+    self.removeListener('destroyed', onDestroyed);
```

Comment:
Hmmm when called directly I assumed `once` would remove the listener.. however the listener should be removed when the function is called for `destroy`. I'll correct this in the next PR.